scripts/config/cspell/cSpell.json

```diff
@@ -26,7 +26,25 @@
     "softwareTerms",
     "scientific-terms-us"
   ],
-  "words": ["casteist", "doxxing"],
+  "words": [
+    "casteist",
+    "doxxing",
+    "abits",
+    "bbase",
+    "countsint",
+    "bmeas",
+    "bbits",
+    "agoodbits",
+    "bgoodbits",
+    "ebase",
+    "emeas",
+    "ebits",
+    "teleporting",
+    "Teleporting",
+    "ebitsa",
+    "ebitsb",
+    "pmzero"
```

Comment:
Most of these shouldn't go in the global dictionary. They should be ignored in the local file, as tedious as that is :/